django/db/models/indexes.py

```diff
@@ -11,7 +11,16 @@ class Index:
     # cross-database compatibility with Oracle)
     max_name_length = 30
 
-    def __init__(self, *, fields=(), name=None, db_tablespace=None, opclasses=(), condition=None):
+    def __init__(
+        self,
+        *,
+        fields=(),
+        name=None,
+        db_tablespace=None,
+        opclasses=(),
+        condition=None,
+        include=(),
```

Comment:
Maybe this should be synchronized with the previous cases and also be `None`, along with the related changes in the rest of the `__init__()` method below.